examples/multiple-choice/index.html

```diff
@@ -41,8 +41,8 @@
       [
         {
           question: 'What framework should we start with?',
-          answers: ['React', 'Ember.js', 'Vue'],
-          answer: 1
+          answers: ['React', 'Ember.js', 'Vue', 'Angular', 'jQuery'],
```

Comment:
Yeah, that would be sick! Actually, not only multi option... something like 0 = as many as you want, 1= radios, 2+ = number of options allowed.